templates/artifactory-oss/0/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2'
+
+services:
+
+  artifactory:
+    image: docker.bintray.io/jfrog/artifactory-oss:5.1.3
+    ports:
+     - "8081"
```

Comment:
Need to set outside exposed port. If not, getting random port from the automatic range...
```
--     - "8081"
++     - "8081:8081"
```

Or better, set a rancher-compose HTTP_PORT variable to be configurable at deploy time...
```
--     - "8081"
++     - "${HTTP_PORT}:8081"
```